nucliadb/tests/nucliadb/integration/test_api.py

```diff
@@ -991,14 +993,53 @@ async def test_file_computed_titles_are_set_on_resource_title(
     fed.field = "email"
     fed.title = extracted_title
     bm.file_extracted_data.append(fed)
-    resp = await nucliadb_grpc.ProcessMessage([bm])
-    assert resp.status == OpStatusWriter.Status.OK
+    await inject_message(nucliadb_grpc, bm)
 
     # Check that the resource title changed
     resp = await nucliadb_reader.get(f"/kb/{kbid}/resource/{rid}")
     assert resp.status_code == 200
     assert resp.json()["title"] == extracted_title
 
+    # Now test that if the title is changed on creation, it is not overwritten
+    kbid = knowledgebox
```

Comment:
This is just a test for a previous PR, right?

Reply:
yup, sorry! forgot to commit it before